examples/real_time_enforcer/nat.tf

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+resource "random_pet" "main" {
+  length    = "1"
+  prefix    = "forseti-simple-example"
+  separator = "-"
+}
+
+resource "google_compute_router" "main" {
+  name    = "${random_pet.main.id}"
+  network = "default"
+
+  bgp {
+    asn = "64514"
+  }
+
+  region  = "us-central1"
```

Comment:
We need to use a different region; this is causing a conflict with the `simple_example` tests.